pages/cockpit/quick-start/get-started.adoc

```diff
@@ -0,0 +1,90 @@
+= Get started with Cockpit
+:page-sidebar: cockpit_sidebar
+:page-permalink: cockpit/1.x/cockpit_quickstart_getstarted.html
+:page-folder: cockpit/quickstart
+:page-description: Gravitee.io Cockpit - Get started
+:page-keywords: Gravitee.io, API Platform, API Management, Cockpit, documentation, manual, guide
+
+== Overview
+
+To get started with Cockpit, enter the product URL: https://cockpit.gravitee.io.
+
+First time users can register a new user account. You can also choose to log in with an existing Google or GitHub account.
+
+== Register a new user account
+
+You can skip this section if you intend to log in to Cockpit with a Google or GitHub account.
+
+. Open Cockpit.
+. Click *Register*.
+. Enter your details and click *SIGN IN*.
++
+Cockpit sends you a confirmation email.
+. Click the *Click here* link to confirm your registration.
+. Enter a new password and click *CONFIRM REGISTRATION*.
+. Enter the company details. The company name you enter is the top level of your Cockpit organizational hierarchy (the _account_) after login.
+. Click *FINISH* to complete the registration.
+
+== Log in to Cockpit
+
+. Open Cockpit with the product URL: https://cockpit.gravitee.io.
+. Log in using one of the following methods:
+
+* Enter your user credentials, then click *SIGN IN*.
+* Click one of the pre-configured identity providers. If this is your first time signing in:
+
+.. Enter the details of the identity provider linked account.
+.. Enter the company details and human readable ID. The company name you enter is the top level of your Cockpit hierarchy (the _account_) after login.
+.. Click *FINISH*.
+
+=== Dashboard
+
+After login, Cockpit opens the dashboard, which displays an interactive map of the organizational hierarchy associated with your account.
+
+image::cockpit/cockpit-dashboard.png[]
+
+In the dashboard map, you can:
+
+* view the dashboard for an account or organization by clicking its dashboard icon image:icons/dashboard-icon.png[role="icon"]
+* expand or collapse the sub-entities for an account or organization by clicking its expand-collapse icon image:icons/expand-collapse-icon.png[role="icon"]
+* update the settings for an environment by clicking its settings icon image:icons/cockpit-settings-icon.png[role="icon"]
+
```

Comment:
Via the dashboard, you can also log in to an installation by clicking on the door icon in installation card
![screenshot-localhost_4004-2021 02 09-14_03_19](https://user-images.githubusercontent.com/1655950/107367446-99789580-6adf-11eb-9aeb-45ec25e19b4b.png)

When you login to an installation via the dashboard, it will open AM / APIM console, scoped in the first environment linked (if many environments are linked to the same installation)